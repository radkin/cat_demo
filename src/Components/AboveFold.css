.picContainerLeft {
  width: 25%;
  float:left;
  width: 18%;
  height: 18%;
}

.picContainerRight {
  width: 25%;
  float: right;
  padding-right: 200px;
  width: 26%;
  height: 26%;
}

.cats {
  display: inline-block;
  font-size: 30px;
}

.fur {
  display: inline-block;
  font-size: 20px;
}

.silly {
  display: inline-block;
  font-size: 15px;
}

.foldParent {
  text-align: center;
}
