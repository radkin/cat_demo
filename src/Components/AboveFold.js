// You can live edit this code below the import statements
import React from 'react';
import Fade from 'react-reveal/Fade';
import rrConfig from 'react-reveal/globals';
import './AboveFold.css';
import onPerch from '../assets/images/on_perch_sm.jpg';
import whatThat from '../assets/images/what_was_that_sm.jpg';

rrConfig({ ssrFadeout: true });

class AboveFold extends React.Component {
  render() {
    return (
      <div className="foldParent">
        <div className="picContainerLeft">
          <Fade left delay={400}>
            <img src={onPerch} alt="on perch" className="pic"/>
          </Fade>
        </div>
        <Fade bottom delay={800}>
          <div className="cats">
            <h1>Cats</h1>
          </div>
        </Fade>
        <Fade bottom delay={1200}>
          <div className="fur">
            <h2>more fur than ever</h2>
          </div>
        </Fade>
        <Fade bottom delay={1500}>
          <div className="silly">
            <h2>Certainly way more silly than you or I.</h2>
          </div>
        </Fade>
        <div className="picContainerRight">
          <Fade right delay={200}>
            <img src={whatThat} alt="what was that" className="pic"/>
          </Fade>
        </div>
      </div>
    );
  }
}

export default AboveFold;
