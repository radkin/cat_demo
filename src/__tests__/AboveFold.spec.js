import React from 'react';
import renderer from 'react-test-renderer';
import { mount } from 'enzyme';

import AboveFold, { Counter, dataReducer } from '../Components/AboveFold';

describe('AboveFold', () => {

  test('snapshot renders', () => {
    const component = renderer.create(<AboveFold />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });

});
