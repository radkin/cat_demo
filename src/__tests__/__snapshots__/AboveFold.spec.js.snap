// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AboveFold snapshot renders 1`] = `
<div
  className="foldParent"
>
  <div>
    <div
      className="react-reveal sky"
      style={
        Object {
          "opacity": undefined,
        }
      }
    >
      <h1>
        Skyline
      </h1>
      <div
        className="react-reveal pr"
        style={
          Object {
            "opacity": undefined,
          }
        }
      >
        <h2>
          scientific solutions
        </h2>
        <div
          className="react-reveal in"
          style={
            Object {
              "opacity": undefined,
            }
          }
        >
          <h2>
            to expand your mind
          </h2>
        </div>
      </div>
    </div>
  </div>
</div>
`;
